src/index.html

```diff
@@ -7,6 +7,102 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="calendar">
```

Comment:
Why do you need block `calendar` in `calendar`? leave only 1

also this block should have modifiers 
`calendar--start-day-{day_name}` and `calendar--month-length-{n}`
![image](https://github.com/mate-academy/layout_calendar/assets/55200530/37733da0-f4e6-4d2f-bfe6-66459cbf9763)

and depends on it calendar should have start day and days count